runtime/src/lib.rs

```diff
@@ -143,15 +143,20 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     transaction_version: 1,
 };
 
-pub const MILLISECS_PER_BLOCK: u64 = 6000;
+pub const MILLISECS_PER_BLOCK: u64 = 12000;
 
 pub const SLOT_DURATION: u64 = MILLISECS_PER_BLOCK;
 
+pub const EPOCH_DURATION_IN_BLOCKS: BlockNumber = 10 * MINUTES;
```

Comment:
why not same as our standalone chain https://github.com/DataHighway-DHX/node/blob/master/pallets/primitives/src/constants/mod.rs#L39

Reply:
done